app/views/content/train-to-be-a-teacher/what-is-a-pgce.md

```diff
@@ -37,17 +37,19 @@ expander:
     link_url: /non-uk-teachers/non-uk-qualifications
 ---
 
-A postgraduate certificate in education (PGCE) is an academic qualification. You can get this through teacher training.
+If you have a degree and are interested in gaining a qualification in education, you may want to do a postgraduate certificate in education (PGCE).
 
-You do not need one to be a qualified teacher in England, but it can help you develop your teaching skills and knowledge.
+You can get a postgraduate certificate in education (PGCE) through teacher training.
```

Comment:
```suggestion
You can get a postgraduate certificate in education through teacher training.
```
Do we need the acronym again here? It's in the sentence before? Or should we just say PGCE in this sentence rather than writing it out in full again?